### PR TITLE
3djc/horus fix #4041

### LINKED ISSUE #4041: Screen Problem in the Folder Models
I copy a ready made model Polizia in a folder Heli. Than I changed some data in the mixer, make a new name DARIO 600 PRO D8 prepare all. After you will load again a model from the folder Heli (in my example) you see a wrong sreen from that model DARIO 600 Pro D8. And you cannot read the name in the screen. See pics.

### PR DESCRIPTION
**Pull request by 3djc**
image

### CODE REVIEW

**Comment by bsongis** [comment]
Did you check it's still OK with a scrollbar on categories?

File: radio/src/gui/480x272/model_select.cpp [hunk 1]
```diff
@@ -498,9 +498,9 @@ bool menuModelSelect(event_t event)
   // Footer
   lcd->drawBitmap(5, LCD_H-FH-20, modelselSdFreeBitmap);
   uint32_t size = sdGetSize() / 100;
-  lcdDrawNumber(22, LCD_H-FH-21, size, PREC1, 0, NULL, "GB");
-  lcd->drawBitmap(80, LCD_H-FH-20, modelselModelQtyBitmap);
-  lcdDrawNumber(105, LCD_H-FH-21, modelslist.modelsCount);
+  lcdDrawNumber(22, LCD_H-FH-21, size, PREC1|SMLSIZE, 0, NULL, "GB");
+  lcd->drawBitmap(70, LCD_H-FH-20, modelselModelQtyBitmap);
+  lcdDrawNumber(92, LCD_H-FH-21, modelslist.modelsCount,SMLSIZE);
```
Comment (bsongis) [inline radio/src/gui/480x272/model_select.cpp]: Space after the comma

**Comment by 3djc** [comment]
Good catch, checking it !

**Comment by 3djc** [comment]
image

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**